src/Illuminate/Foundation/Bootstrap/LoadConfiguration.php

```diff
@@ -66,9 +66,10 @@ protected function loadConfigurationFiles(Application $app, RepositoryContract $
     protected function getConfigurationFiles(Application $app)
     {
         $files = [];
+        $path = realpath($app->configPath());
```

Comment:
https://github.com/laravel/laravel/blob/master/bootstrap/app.php#L15

It's already using `realpath()`, any reason to check it again?